cmd/neofs-cli/internal/key/raw.go

```diff
@@ -32,23 +27,16 @@ func Get(cmd *cobra.Command) *ecdsa.PrivateKey {
 
 func get() (*ecdsa.PrivateKey, error) {
 	keyDesc := viper.GetString(commonflags.WalletPath)
-	priv, err := keys.NewPrivateKeyFromWIF(keyDesc)
-	if err == nil {
-		return &priv.PrivateKey, nil
-	}
-
-	p, err := getKeyFromFile(keyDesc)
-	if err == nil {
-		return p, nil
-	}
-
-	w, err := wallet.NewWalletFromFile(keyDesc)
-	if err == nil {
-		return FromWallet(w, viper.GetString(commonflags.Account))
-	}
+	if _, err := os.Stat(keyDesc); err == nil {
```

Comment:
Won't we lose FS errors here and always return `invalid key`?

Reply:
If we have only wallet/raw key left, both a file, and read-error should be the same as stat one.

Reply:
Thanks, done.